lib/chef/knife/core/bootstrap_context.rb

```diff
@@ -69,7 +69,8 @@ def trusted_certs
 
         def config_content
           client_rb = <<-CONFIG
-log_location     STDOUT
+log_level        :#{@chef_config[:config_log_level]}
+log_location     "#{@chef_config[:config_log_location]}"
```

Comment:
There are two possible issues that may require fix:
1. What happens if `config_log_level` and `config_log_location` are not passed? Are the values of `log_level` and `log_location` getting passed in that case?
2.  Is `config_log_location  STDOUT` getting passed successfully?